pyblish_lite/control.py

```diff
@@ -507,6 +544,26 @@ def __init__(self, parent=None):
         right_view.customContextMenuRequested.connect(
             self.on_plugin_action_menu_requested)
 
+        for box in (show_errors,
+                    show_records,
+                    show_debug,
+                    show_info,
+                    show_warning,
+                    show_error,
+                    show_critical):
+            box.setChecked(True)
+
+        artist_tab.setChecked(True)
+
+    def on_item_expanded(self, index, state):
+        if not index.data(model.IsExpandable):
+            return
+
+        if state is None:
+            state = not index.data(model.Expanded)
+
+        # index.model().setData(index, state, model.Expanded)
```

Comment:
Uncomment this line to expand items in the Terminal and inspect their properties; it currently doesn't layout correctly and doesn't elide text.